app/config/question-bank.js

```diff
@@ -1702,7 +1702,7 @@ const questionBank = {
             <p class="govuk-body">This grant amount combines:<p>
             <ul>
             <li>£{{_calculatedGrantCalf_}} for calf housing costs (40% of £{{_CalfHousingCost_}})</li>
-            <li>£{{_calculatedGrantSolar_}} for solar PV costs (25% of £{{_SolarPVCost_}} is £{{_calculatedGrantSolarPreCap_}}, capped at £{{_solarCap_}})</li>
+            <li>£{{_calculatedGrantSolar_}} for solar PV costs (25% of £{{_SolarPVCost_}} is £{{_calculatedGrantSolarPreCap_}}, capped at £{{_calculatedGrantSolar_}})</li>
             </ul>
             <p class="govuk-body">As calf housing costs take grant funding priority, you may be able to apply for a grant of up to £{{_solarCap_}} for solar PV system costs. The maximum grant is £500,000.`,
```

Comment:
solarCap here needs to be replaced with calculatedGrantSolar as well

Reply:
done